internal/storage/bundle/bundle.go

```diff
@@ -207,6 +208,11 @@ func (b *Bundle) loadPolicySet(idHex, fileName string) (*runtimev1.RunnablePolic
 		return nil, fmt.Errorf("failed to unmarshal %s: %w", idHex, err)
 	}
 
+	err = compile.MigrateCompiledPolicies(rps)
```

Comment:
As far as I understand, there is no caching for binary stores unlike source stores where the compiled policies are cached. I feel like it might be nice to introduce a cache here so that we don't have to read from the `bundleFS`, unmarshal, and run the migration functions every time we load a policy.